public/Aryan Suri/Citation/attribution.js

```diff
@@ -12,7 +12,8 @@ function attribution() {
         }
         let titlestr = `"${title}"`;
         let pageID = $("#pageIDHolder").text();
-        let url = `https://chem.libretexts.org/@go/page/${pageID}`;
+        let currentLibrary = window.location.host.split('.')[1] ? window.location.host.split('.')[0] : 'chem';
```

Comment:
FYI you can also rely on the `LibreTexts.parseURL()` function to get the subdomain and/or path.